stl/inc/xpolymorphic_allocator.h

```diff
@@ -13,6 +13,10 @@
 #include <utility>
 #include <xmemory>
 
+#if _HAS_CXX20
+#include <memory>
+#endif // _HAS_CXX20

```

Comment:
Only `std::destroy_at()`.
I think that would be preferable.
But I'm not sure how to move it (mainly, where?)

Reply:
That's a good question - we generally try to follow the Standard's order of declarations when there isn't a reason otherwise - however that doesn't provide much of a guide here. While trying to look for related machinery, I realized that #1313 (created today) is changing `std::destroy_at()`'s definition - so to avoid merge conflicts, let's leave this code as-is for now. We can refactor to improve throughput in the future.

Reply:
I see. Yes, sir!

Reply:
I think the official response is one of "Meow", "Hiss" and "Purr"